lib/bindings.js

```diff
@@ -21,6 +21,9 @@ const loadBinary = function (baseName) {
   var binding = null;
 
   nodeFiles.forEach((file) => {
+    // brute force try to load the node extension across operating systems and versions, use whichever loads at all
+    // TO be researched: why not use https://nodejs.org/api/process.html#process_process_platform and why 
```

Comment:
If the naming pattern would use the platform  (linux vs. win32 vs. darwin) the code would have to only brute force try the nodejs versions but not everything.